### DISCUSSION TITLE
Links legais

### DISCUSSION THREAD

**Discussion by lavratti** [lavratti/tcc-rem#2] [Ideas]
Esse link é bacana

**Comment by lavratti** [comment]
https://github.com/keunwoochoi/MSD_split_for_tagging

**Comment by lavratti** [comment]
https://github.com/keunwoochoi/music-auto_tagging-keras
Paper: https://arxiv.org/pdf/1609.04243.pdf

**https://github.com/keunwoochoi/music-auto_tagging-keras/blob/master/music_tagger_crnn.py**

**Comment by lavratti** [comment]
https://github.com/ZainNasrullah/music-artist-classification-crnn
**https://github.com/ZainNasrullah/music-artist-classification-crnn/blob/master/src/models.py**

_It should be noted that training each model is computationally expensive and can take several hours even with reliable hardware. At minimum, a Nvidia GTX 1080 GPU is recommended with at least 16GB of memory on the machine_

**Comment by RafaeloXIII** [comment]
https://github.com/meetps/crnn-music-genre-classification
https://github.com/ZainNasrullah/music-artist-classification-crnn
https://github.com/sharathadavanne/sed-crnn
